src/components/AddDataInstructions.tsx

```diff
@@ -11,8 +14,69 @@ export const AddDataInstructions = ({
 }: AddDataInstructionsProps) => (
   <Panel>
     <PanelBody>
-      <p>TODO: add data instructions</p>
-      {children}
+      <article className="add-data-instructions">
+        <h3>Add data to the PDC</h3>
+
+        <p>
+          <b>Adding data via the bulk uploader</b>
+          {' '}
+          is drag-and-drop easy:
+        </p>
+
+        <ol>
+          <li>
+            Create a CSV of your data with base field keys as the first row
+            (see table below). If you want, you can start with
+            {' '}
+            <a href={bulkUploadTemplateUrl.toString()}>
+              this template of recommended base fields
+            </a>
+            .
+          </li>
+          <li>
+            Upload your CSV in the sidebar to the left.
+          </li>
+        </ol>
+
+        <p>
+          Note that you cannot create new base fields via the bulk uploader.
```

Comment:
In the spirit of "humans are better at being told what to do than what not to do" I wonder if this would be better written as "you may only use the base fields listed below"

The idea of new base fields may or may not actually be on the mind of the user, and I think answering that might be cleaner as a totally independent use case some day in the future (and in either event, this section is not addressing that use case / explaining how to do it!)

Reply:
That makes sense to me, and I agree with it - good catch! Unfortunately, I think for now I'm going to merge as-is and follow up with @reefdog when he's back.

Reply:
Good thought. Opened as #464.